manage_sections/tests/test_view_create_section_form.py

```diff
@@ -153,3 +154,16 @@ def test_section_form_view_status_without_custom_canvas_course_id(self, log_repl
         request.LTI['lis_course_offering_sourcedid'] = None
         create_section_form(request)
         render.assert_called_with(request, 'manage_sections/error.html', status=500)
+
+    @patch('manage_sections.views.logger.error') # Mock the logger to keep log messages off the console.
+    @patch('lti_permissions.decorators', is_allowed=Mock(return_value=False))
+    def test_section_form_view_when_not_permitted(self, lti_decorator, log_replacement, render):
+        """
+        When the user does not have the right permissions, verify that it returns unauthorized
```

Comment:
Can we clarify the comment? 302 is a redirect, not an unauthorized HTTP code. I think we want to check that the 302 redirects to the unauthorized page, right?